src/compiler/GF/Command/CommonCommands.hs

```diff
@@ -170,10 +171,14 @@ commonCommands = fmap (mapCommandExec liftSIO) $ Map.fromList [
        restrictedSystem $ syst ++ " <" ++ tmpi ++ " >" ++ tmpo
        fmap fromString $ restricted $ readFile tmpo,
        -}
-       fmap fromString . restricted . readShellProcess syst $ toString arg,
+       fmap (fromStrings . lines) . restricted . readShellProcess syst . unlines . map (dropWhile (=='\n')) $ toStrings $ arg,
+
      flags = [
        ("command","the system command applied to the argument")
        ],
+     options = [
+       ("lines","preserve input lines, and return output as a list of lines")
+       ],
```

Comment:
Why is this here? Does it do anything? The point was to have the default behaviour to have lines, not to add a flag.